src/redux/store.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import bookReducer from './books/booksSlice';
+
+const store = configureStore({
+  reducer: {
+    books: bookReducer,
+  },
+});
+
```

Comment:
- [ ] Please combine both reducers into the root reducer as requested in the project requirements. You should import the categories reducer and combine that here with the books reducer.